src/System/include/BipedalLocomotion/System/StdClock.h

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file StdClock.h
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_SYSTEM_STD_CLOCK_H
+#define BIPEDAL_LOCOMOTION_SYSTEM_STD_CLOCK_H
+
+#include <BipedalLocomotion/System/IClock.h>
+
+#include <chrono>
+
+namespace BipedalLocomotion
+{
+namespace System
+{
+
+/**
+ * StdClock implements a the IClock interface using.
+ * The clock can be easily used as follows
+ * \code{.cpp}
+ * #include <BipedalLocomotion/System/Clock.h>
+ *
+ *
+ * auto start = BipedalLocomotion::clock().now();
+ * foo();
+ * auto end = BipedalLocomotion::clock().now();
+ * std::chrono::duration<double, std::milli> elapsed = end-start;
+ * \endcode
+ */
+class StdClock final : public IClock
+{
+public:
+    /**
+     * Get the system current time
+     * @return the current time since epoch computed with std::chrono::system_clock
```

Comment:
```suggestion
     * @return the current time in milliseconds since epoch computed with std::chrono::system_clock
```

Reply:
Are you sure that are milliseconds?

Reply:
Not completely. That is what I assumed by reading https://github.com/dic-iit/bipedal-locomotion-framework/blob/914faefe0b1567bdddfe72c8b0dd178b09ec9190/src/System/include/BipedalLocomotion/System/StdClock.h#L30
but this is also the reason why I proposed the modification :grin:

Reply:
I think I might have misinterpreted indeed. It probably deserves an additional line of documentation

Reply:
Hi @S-Dafarra, from c++ documentation 
```cpp
template<    class Rep,    class Period = std::ratio<1>> 
class duration; 
```
It consists of a count of ticks of type Rep and a tick period, where the tick period is a compile-time rational constant representing the number of seconds from one tick to the next.
Notice that a `std::ratio` represents the tick period (i.e. the number of seconds per tick).
So if you write
```cpp
auto now = BipedalLocomotion::clock().now();
```
you will have a `chrono::duration` whose ticks are expressed in seconds. (i.e. if you print `now.count()` you will have the seconds from epoch)
However std::chrono::duration can be construct also with the following constructor
```cpp
template< class Rep2, class Period2 >
constexpr duration( const duration<Rep2,Period2>& d );
```
That is the one used in the example
```cpp
std::chrono::duration<double, std::milli> elapsed = end-start; 
```
Here `std::milli` is `Period2`.

In conclusion, the function returns a class whose number of seconds per tick is equal to 1. 
I can add the following line in the documentation
```
* @note `BipedalLocomoition::clock().now().count()` returns the number of seconds since epoch 
```

Reply:
@S-Dafarra let me know if you're fine with https://github.com/dic-iit/bipedal-locomotion-framework/pull/269/commits/005d7c158d327cef65a38c8d878e7ebf20818898

Reply:
I am! Thanks!